lib/lookup.json

```diff
@@ -329,6 +329,13 @@
     "skip": ["win32"],
     "head": true
   },
+  "openid-client": {
+    "maintainers": ["panva"],
+    "prefix": "v",
+    "scripts": ["tap:node"],
+    "skip": ["aix", "<20"],
+    "comment": "esbuild doesn't support aix"
```

Comment:
esbuild _does_ support AIX, via https://github.com/evanw/esbuild/pull/3550 from esbuild@0.19.10: https://github.com/evanw/esbuild/issues/3549#issuecomment-1861911942

Reply:
I just copied the block from jose... might need updating too

Reply:
lmk if i should that in this PR or another one